src/AxiomTest.sol

```diff
@@ -129,6 +166,50 @@ abstract contract AxiomTest is Test {
         vm.makePersistent(axiomV2QueryAddress);
     }
 
+    /// @dev Set up crosschain Axiom contracts
+    /// @param forkBlock The block number that the fork was created from
+    /// @param sourceChainId The chain ID of the source chain
+    /// @param isBlockhashOracle Whether the bridging is via a blockhash oracle
+    /// @param bridgeId The ID of the bridge
+    function _setupAxiomCrosschainFromFork(
+        uint256 forkBlock,
+        uint64 sourceChainId,
+        bool isBlockhashOracle,
+        uint8 bridgeId
```

Comment:
do we want to stub out the broadcaster/bridgeId logic as well or keep it clean for now?

Reply:
I think we can leave it unimplemented for now.